docs/threat-modeling-for-designers/threat-modeling-for-designers.md

```diff
@@ -0,0 +1,57 @@
+# Threat Modeling for Designers
+
+This is a living, work-in-progress documentation to shape how might _Threat Modeling_ (borrowing from security field) look like for designers. Alternatively, we can relabel "threat modeling" as "risk assessment."
+
+We welcome collobration on this documentation — Suggest and create a PR to add, suggest improvement, etc.
+
+## Why
+
+As a framework, _threat modeling for designers_ will help designers:
+
+1. Understand the threats users face and how designers can counter those threats
```

Comment:
I agree with the feedback above about `threat modeling` being a new term for many designers, and I think `threat` is also a new idea. What about....

1. Understand the threats users face in design, and how designers can identify and prevent harm to their users.

Reply:
Updated based on Christine's suggestion (TY!)